packages/plugins/plugin-store/src/plugin.svelte

```diff
@@ -0,0 +1,17 @@
+<svelte:options tag={null} />
+
+<script lang="ts">
+import { PluginStore } from '@oscd-plugins/uilib/src/lib/plugins/plugin-store'
+import * as pckg from '../package.json'
```

Comment:
Don't import everything from package, this file specifically doesn't seem to use anything external, no need for this line